TASK/SetHighFallTask.md

```diff
@@ -5,16 +5,26 @@ ns: TASK
 
 ```c
 // 0x8C825BDC7741D37C 0xBBB26172
-void SET_HIGH_FALL_TASK(Ped ped, Any p1, Any p2, Any p3);
+void SET_HIGH_FALL_TASK(Ped ped, int p1, Any p2, Any p3);
 ```
 
 ```
-Makes the ped ragdoll like when falling from a great height  
+Makes the ped ragdoll like when falling from a great heigh.
+Playing with p1 and p2 changes the animation's style and behavior.
+In most cases the ped moves backwards then fall on the ground or collapse with a very small change to it's position.
+If p1 or p2 is 1 the ped will move backwards  
```

Comment:
You mention p1 is duration, but this does not suggest that. Could you elaborate on this?